frontend/src/lib/Buttons.ts

```diff
@@ -266,6 +278,19 @@ export default class Buttons {
     );
   }
 
+  private _terminateRun(ids: string[], useCurrentResource: boolean,
+    callback: (_: string[], success: boolean) => void): void {
+    this._dialogActionHandler(
+      ids,
+      'Do you want to terminate this run? This action cannot be undone.',
```

Comment:
I think this text can use some more elaboration. Maybe it should indicate what happens to running pods.

Reply:
How about:
`Do you want to terminate this run? This action cannot be undone. The run is terminated by setting the activeDeadlineSeconds of the Argo workflow to 0. This terminates the workflow without deleting the pods.`

Reply:
> The run is terminated by setting the activeDeadlineSeconds of the Argo workflow to 0. This terminates the workflow without deleting the pods.

I'd say that's too much implementation info.

Reply:
How about
"Do you want to terminate this run?
All the running tasks will be terminated."

Reply:
I agree, there's no need to surface Argo here.
Do running tasks (pods) really terminate? Argo just doesn't schedule any more pods AFAIR.

Reply:
Yes, the pods are terminated:
https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#job-termination-and-cleanup

Reply:
Is Argo using a k8s job? I didn't think so. Did you verify that any long running pods are killed?

Reply:
Pods are terminated before they complete, not after